src/applications/journal_display/journal_printer.c

```diff
@@ -95,694 +96,694 @@ static const WindowTemplate Unk_ov81_021D3508[] = {
     { 0x1, 0x4, 0x4, 0x1B, 0x12, 0xF, 0x31 }
 };
 
-void ov81_021D1610(UnkStruct_ov81_021D1610 *param0)
+void JournalPrinter_AddWindows(JournalManager *journalManager)
 {
     u32 i;
 
     for (i = 0; i < 4; i++) {
-        Window_AddFromTemplate(param0->bgConfig, &param0->window[i], &Unk_ov81_021D3508[i]);
+        Window_AddFromTemplate(journalManager->bgConfig, &journalManager->window[i], &Unk_ov81_021D3508[i]);
     }
 }
 
-void ov81_021D1634(UnkStruct_ov81_021D1610 *param0)
+void JournalPrinter_RemoveWindows(JournalManager *journalManager)
 {
     u32 i;
 
     for (i = 0; i < 4; i++) {
-        Window_Remove(&param0->window[i]);
+        Window_Remove(&journalManager->window[i]);
     }
 }
 
-void ov81_021D164C(UnkStruct_ov81_021D1610 *param0, u32 param1)
+void JournalPrinter_PrintEntry(JournalManager *journalManager, u32 param1)
 {
-    Window *v0;
-    Window *v1;
+    Window *window1;
+    Window *window2;
```

Comment:
Rename to something more meaningful that expresses how the two windows differ (or leave them as `v0` and `v1` for now if you can't tell).